### ISSUE TITLE
Create a Basic Website with HTML, CSS

### ISSUE THREAD

**Issue by tamalchowdhury** [tamalchowdhury/bankcardsbangladesh#11]
Hello Dev, please build a basic website for this project.

In the website add a header, and a footer. Then in the main body, display the current bank info in the `README.md` file as simple table or what you find suitable.

Add some basic CSS styling.

We will convert the site to React in the upcoming revisions. Right now the goal is to publish this site on github.

To build this site, make a fork of this repo. Then in your coding environment, do the git linking setup for upstream master branch. Please take a look at this [article](https://tamalchowdhury.com/setup-git-remote-upstream-master-connection-for-open-source-contribution) + [video](https://egghead.io/lessons/javascript-how-to-fork-and-clone-a-github-repository) on how to do this.

Add the website files (index.html, style.css) in the root of this project and send the pull request.

Let me know if you need any help.

This task will count as one valid contribution for the Hacktoberfest. 

All the best!

**Comment by Pritam-2002** [comment]
On it..

**Comment by Pritam-2002** [comment]
#12 done with the changes

**Comment by Pritam-2002** [comment]
@tamalchowdhury what about the next changes ???is it be good if it is hosted on any platform?

**Comment by tamalchowdhury** [comment]
Okay, just saw the website. We have a lot of design work to do to actually publish it and be proud of it. Some CSS layout design will make it better.

This is after merging #12 

<img src="https://i.postimg.cc/6QfRpB8m/bank-card-from-pritom.png" alt="bank cards bd first version"/>

I will publish the page to github pages for now.

**Comment by Pritam-2002** [comment]
it is not responsive yet we have to make it  fully responsive

**Comment by tamalchowdhury** [comment]
@Pritam-2002 are you able to make a modern design, with some Sans fonts? If so, please create a new design, add some text (copy from the readme), perhaps a picture, responsive, in the next PR. Go ahead.

Thank you for the work.

**Comment by Pritam-2002** [comment]
@tamalchowdhury yes i can make this a modern website ....with some  cool animation effects On it ....i think it would be helpful for me to discuss it in linkedin for making the changes ?

**Comment by tamalchowdhury** [comment]
No need any animation for now. Continue all project discussions here for
future reference.

**Comment by Pritam-2002** [comment]
Ok..on it

**Comment by Pritam-2002** [comment]
No animation just modern design right?  pictures and hover effects?..is it
ok

**Comment by tamalchowdhury** [comment]
This is great work @farhanajuli, this is taking shape!

[![bank-cards-from-Farhana.png](https://i.postimg.cc/7hzXwHbR/bank-cards-from-Farhana.png)](https://postimg.cc/ZvT6Lh8P)

**Comment by Pritam-2002** [comment]
So the ui will be same or some modern changes?? Basically I am working on
the website in  more modern changes...

**Comment by tamalchowdhury** [comment]
@Pritam-2002 Farhana has already made some changes to the website. Perhaps you copy your code in a different folder, then pull the changes from main branch, and then implement your new design on top of it.

Thanks
